zio-interop-cats/shared/src/main/scala/zio/interop/cats.scala

```diff
@@ -165,7 +167,7 @@ sealed abstract class CatsInstances2 {
 }
 
 private class CatsDefer[R, E] extends Defer[ZIO[R, E, *]] {
-  def defer[A](fa: => ZIO[R, E, A]): ZIO[R, E, A] = ZIO.suspendSucceed(fa)
+  def defer[A](fa: => ZIO[R, E, A]): ZIO[R, E, A] = ZIO.suspendSucceed(fa)(CoreTracer.newTrace)
```

Comment:
`fa` has a source attached because it is a `Function0`.

Reply:
fixed